grakn-graph/src/main/java/ai/grakn/graph/internal/TypeImpl.java

```diff
@@ -150,6 +150,7 @@ V addInstance(Schema.BaseType instanceBaseType, BiFunction<Vertex, T, V> produce
         }
 
         Vertex instanceVertex = getGraknGraph().addVertex(instanceBaseType);
+        getGraknGraph().getConceptLog().addedInstance(getLabel());
```

Comment:
This is really petty: but if the behaviour for decrementing the count is in `InstanceImpl`, then that's also where the increment behaviour should be, not `TypeImpl`.